mindmaps-graql/src/main/java/io/mindmaps/graql/internal/analytics/Analytics.java

```diff
@@ -46,6 +46,7 @@
 public class Analytics {
 
     public static final String keySpace = "mindmapsanalyticstest";
+    public static final String TYPE = "type";
```

Comment:
[QUESTION] Why is this hardcodded to "type" ? If this is the meta type then you should be using DataType.ConceptMeta.TYPE.getId();